src/metrics/mendel_metrics.nit

```diff
@@ -1,6 +1,9 @@
 # This file is part of NIT ( http://www.nitlanguage.org ).
 #
-# Copyright 2014 Alexandre Terrasa <alexandre@moz-code.org>
+# Copyright 2014-2015 Alexandre Terrasa <alexandre@moz-code.org>
+# Copyright 2014      Jean Privat <jean@pryen.org>
+# Copyright 2014-2016 Jean-Christophe Beaupré <jcbrinfo@users.noreply.qithub.com>
+# Copyright 2016      Lucas Bajolet <r4pass@hotmail.com>
```

Comment:
Since about 2012, we don't explicitly declare our copyright in each file as the info is available with git.